### PR TITLE
Banner update.

### PR DESCRIPTION
**Pull request by mikelehen**

### CODE REVIEW

File: src/components/Analytics/utils.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export const legacyTracker: Tracker = {
 export enum EventCategory {
   API = 'api',
   ARTICLES = 'articles',
+  BANNER = 'banner',
```
Comment (mikelehen) [inline src/components/Analytics/utils.ts:37]: Is this a good idea?  I wasn't sure what category to use.  It seems in the past we've created new categories for banners (e.g. VOTE_2020).  But I wasn't sure.

File: src/components/Banner/SpringSurgeBanner.style.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  }
+`;
+
+export const InnerContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  a {
```
Comment (mikelehen) [inline src/components/Banner/SpringSurgeBanner.style.tsx:25]: This whole file is copied from VaccinationBanner.style.tsx except I dropped all the button stuff I didn't need and added this styling for links...

File: src/components/Banner/SpringSurgeBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import { Wrapper, InnerContainer, Body } from './SpringSurgeBanner.style';
+import { scrollWithOffset } from 'components/TableOfContents';
+import { HashLink } from 'react-router-hash-link';
+
+const trackClick = () => {
+  trackEvent(
+    EventCategory.BANNER,
+    EventAction.CLICK,
+    'Banner: Hospitalizations explore chart',
+  );
```
Comment (mikelehen) [inline src/components/Banner/SpringSurgeBanner.tsx:12]: This all seem reasonable?

**Review by pnavarrc — COMMENTED** [review]
Added some notes, will test after the category change!

File: src/components/Analytics/utils.ts (hunk 1 above)
Comment (mikelehen) [reply src/components/Analytics/utils.ts]: changed to HOMEPAGE_BANNER

File: src/components/Banner/SpringSurgeBanner.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+};
+
+const VaccinationsBannerInner: React.FC = () => {
```
[the remark this answers is not in the document]
Comment (mikelehen) [reply src/components/Banner/SpringSurgeBanner.tsx]: Ooops!  Too much copy/paste. 😬   Fixed.

**Review by mikelehen — COMMENTED** [review]
Updated, thanks!

**Review by pnavarrc — APPROVED** [review]
Looks good!

**Pull request merged**